src/stylesheets/components/_example.scss

```diff
@@ -71,5 +71,4 @@
 
 .app-example__code {
   position: relative;
-  @include govuk-font($size: 19);
```

Comment:
Mind if we keep `font-weight: 400`?

It's changed the weight of the code examples and (maybe) we should restore these too?

```css
-webkit-font-smoothing: antialiased;
-moz-osx-font-smoothing: grayscale;
```

Reply:
If it's also missing `govuk-typography-common` then I'll just put it back in

Reply:
It is, but we use a code font-family here so doesn't need all of it

We should probably get better at setting things on `.app-example` and letting child elements inherit?

So both `.app-example__code` and `.app-example__new-window` just have the bare minimum set

Reply:
I've added `govuk-font($size: 16` to `app-example`. This means that `code` and `new-window` inherit what they need and we can get rid of `app-example__new-window` since everything it specifically needs is cascading down. That means the `pre` is still having to override the font family which feels a bit ham-fisted but better than it was. Lemme know what you think.